new-dti-website/components/team/MemberDisplay.tsx

```diff
@@ -0,0 +1,101 @@
+import { useRef, useState } from 'react';
+import MemberGroup from './MemberGroup';
+import Icon from '../icons';
+import FA23Members from '../../../backend/src/members-archive/fa23.json';
+import teamRoles from './data/roles.json';
+import roleIcons from './data/roleIcons.json';
+import { populateMembers } from '../../src/app/utils';
+
+type DisplayCategory = 'Full Team' | 'Leads' | 'Design' | 'Product' | 'Business' | 'Development';
```

Comment:
Delete this type and use `string` instead, since we're reading these values directly from the JSON anyways. So we're not benefitting from any type-checking going on here.